### PR TITLE
Adding update for redis lists

### PR DESCRIPTION
**Pull request by tbickford**
This pertains to the PR I have made for adding the optional authentication params for the Redis Scaler (https://github.com/kedacore/keda/pull/962).  Added some additional wording around the use of those attributes as env vars. I added this to the `2.0` version and created a new version `1.6` as I would think it would go out with the next release? Is there another branch I should be using as the base besides the master branch for potential adds in the next release?

### CODE REVIEW

File: content/docs/2.0/scalers/redis-lists.md [hunk 1]
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
 
 You can authenticate by using a password.
 
+**Connection Authentication:**
+
+- `address` - The host and port for the Redis server (host:port format).
+
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:45]:
```suggestion
```

File: content/docs/2.0/scalers/redis-lists.md [hunk 2]
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
...
+- `host` - The host of the Redis server. If specified, the `port` should also be specified
+
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:47]:
```suggestion
```

File: content/docs/2.0/scalers/redis-lists.md [hunk 3]
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
 
 You can authenticate by using a password.
 
+**Connection Authentication:**
+
+- `address` - The host and port for the Redis server (host:port format).
+
+- `host` - The host of the Redis server. If specified, the `port` should also be specified
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:46]:
```suggestion
- `host` - The hostname of the Redis server. If specified, the `port` should also be specified
```

File: content/docs/2.0/scalers/redis-lists.md [hunk 4]
```diff
@@ -39,6 +39,14 @@ The `databaseIndex` parameter let the user select the redis database to use. If
 
 You can authenticate by using a password.
 
+**Connection Authentication:**
+
+- `address` - The host and port for the Redis server (host:port format).
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:44]:
```suggestion
- `address` - The hostname and port for the Redis server (host:port format).
```

**Comment by zroubalik** [comment]
@tomkerkhove @tbickford what is the status of this?

**Comment by tbickford** [comment]
> @tomkerkhove @tbickford what is the status of this?

I thought this was good to go or did I miss something?

**Comment by tomkerkhove** [comment]
I've added some suggestions above but they are nits, for the rest I'm fine.

Up to you folks.

**Comment by tbickford** [comment]
> I've added some suggestions above but they are nits, for the rest I'm fine.
> 
> Up to you folks.

My bad - I had thought I addressed those items. I pushed an update.

**Pull request merged**